rook/workflows/ceph_benchmarks.yml

```diff
@@ -1,17 +1,22 @@
 options:
   env:
     KUBECONFIG: ./kubeconfig/config
+    NAMESPACE: kubestone
 
 steps:
 - id: start-pods
   uses: docker://bitnami/kubectl:1.17.4
-  args: [create, -f, ./ceph_benchmarks/deployment.yml]
+  runs: [bash, -euc]
+  args:
+  - |
+    kubectl create -n "$NAMESPACE" -f ./ceph_benchmarks/deployment.yml
 
 - id: copy-config
   uses: docker://bitnami/kubectl:1.17.4
   runs: [bash, -euc]
   args:
   - |
+    while [ "$(kubectl get pod -l app=ceph-benchmarks -o jsonpath="{.items[0].status.phase}")" != "Running" ]; do echo "waiting for pod"; done
```

Comment:
couple of things:
  * if `status.phase` gets something else that `Running`, it'll loop indefinitely, so could add a counter and have a max limit of checks. E.g. check 20 times and fail if it didn't succeed
  * we might want to add a `sleep` in between loops so that this doesn't bombard the server

Reply:
I see. Thanks.

Reply:
Since adding the sleep steps would further increase the complexity of the code, can we move the logic to a separate bash script?

Reply:
yeah, that works too. It'd be like 5 more lines of code though, no? like:

```bash
count=0
while ...; do
  if [ count == 10 ]; then
    exit 1
  fi
  kubectl ....
  count=$(( count + 1 ))
  sleep 10
done
```

Reply:
Yeah,  that's true. Not much addition in lines, but maybe it will look bad inline in the workflow. Thanks